src/plugins/irc/irc-command.c

```diff
@@ -5017,6 +5017,8 @@ irc_command_display_server (struct t_irc_server *server, int with_detail)
                 IRC_COLOR_CHAT_DELIMITERS,
                 IRC_COLOR_RESET,
                 (server->temp_server) ? _(" (temporary)") : "",
+                (server->nick) ? " nick: " : "",
```

Comment:
This string should probably be translatable.

Reply:
Sure we could do that. Does it make sense to introduce a new translatable string for this change though? (i'm honestly not sure), and should that be `_(" nick: ")`? And if it should possibly be `nick:`, then the code would be a bit more invovled/ugly.